rules/windows/execution_suspicious_pdf_reader.toml

```
@@ -14,7 +14,64 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious PDF Reader Child Process"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating Suspicious PDF Reader Child Process
+
+PDF is a common file type used in corporate environments and most machines have software to deal with these files. This
```

"Deal with" is a little informal: maybe "handle"? Or "open", "access", "read"?
```suggestion
PDF is a common file type used in corporate environments and most machines have software to handle these files. This
```